src/index.js

```diff
@@ -16,7 +16,11 @@ const unexpectedEnzyme = {
         return reactWrapper && reactWrapper instanceof ReactWrapper;
       },
       inspect: function(reactWrapper, depth, output) {
-        output.appendInspected(reactWrapper.getElement());
+        const inspected = reactWrapper.exists()
+          ? reactWrapper.getElement()
+          : reactWrapper.root().getElement();
+
```

Comment:
When `reactWrapper` does not exist, i.e. because we have something like:

```jsx
const Foo = <div><span id="root"></div>;

expect(Foo.find('#child'), 'to exist));
```

Then we can render `Foo`, it would be great if we could somehow wrap `ReactWrapper` and intercept the `.find` so we could use that on the error message.

Reply:
We can get this error message https://github.com/albertfdp/unexpected-enzyme/pull/3/commits/b1b68cbc40e7d085069553df206167aa2fdcb027#diff-349e536907599e98cd2b54908e099051R3 which is definitely better than `"expected false to be true"` but still not very helpful.